typst-package/manual.typ

```diff
@@ -442,7 +442,7 @@ The following table documents supported values and their #typst-type("str") repr
     stroke: gray.lighten(60%),
     radius: 2pt,
     width: 100%,
-    text(size: 8pt, par(linebreaks: "simple", code-block)),
+    text(size: 8pt, code-block),
```

Comment:
Does `par(linebreaks: "simple", text(size: 8pt, code-block))` work?

Reply:
No. In 0.13, code block is considered as "block level", so it cannot present in par